dashboard/src/components/jobs/parameters/DataParamHandler.vue

```diff
@@ -0,0 +1,159 @@
+      {
+        weather_granularity: "system",
+        irradiance_type: "standard",
+        temperature_type: "cell"
+      }
+    Or:
+      {
+        predicted_data_parameters: {
+          data_available: "weather only",
+          weather_granularity: "system",
+          irradiance_type: "standard",
+          temperature_type: "cell"
+        },
+        actual_data_parameters: {
+          weather_granularity: "system",
+          irradiance_type: "standard",
+          temperature_type: "cell",
+          performance_granularity: "system"
+        }
+      }
+
+-->
+<template>
+  <div class="data-param-handler">
+    <div class="grid-container my-1">
+      <data-params
+        v-for="dataType of requiredDataParams"
+        :key="dataType"
+        :dataType="dataType"
+        :jobClass="jobClass"
+        @new-data-params="setDataParams"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Watch } from "vue-property-decorator";
+
+import DataParams from "@/components/jobs/parameters/DataParams.vue";
+
+Vue.component("data-params", DataParams);
+
+@Component
+export default class DataParamHandler extends Vue {
+  @Prop() jobTypeParams!: Record<string, string>;
+  parameters!: Record<string, any>;
+  data() {
+    return {
+      parameters: {}
+    };
+  }
+
+  mounted() {
+    this.emitParams();
+  }
+
+  setDataParams({
+    type,
+    parameters
+  }: {
+    type: string;
+    parameters: Record<string, string>;
+  }) {
+    if (type == "data_parameters") {
+      // Generic data_parameters type indicates that the data parameters should
+      // be added to the root level job specification.
+      this.parameters = parameters;
+    } else {
+      this.parameters[type] = parameters;
+    }
+    this.emitParams();
+  }
+
+  get isMonthly() {
+    return (
+      "compare" in this.jobTypeParams &&
+      this.jobTypeParams["compare"].includes("monthly")
+    );
+  }
+
+  get jobClass() {
+    if ("compare" in this.jobTypeParams) {
+      return "compare";
+    } else if (
+      this.jobTypeParams["calculate"] == "weather-adjusted performance ratio"
+    ) {
+      return "calculatepr";
+    } else {
+      return "calculate";
+    }
+  }
+  get requiredDataParams() {
+    if (this.isMonthly) {
+      return [];
+    }
+    if (this.jobClass == "calculate") {
+      if (this.jobTypeParams["calculate"] == "predicted performance") {
+        return ["predicted"];
+      } else {
+        return ["expected"];
+      }
+    } else if (this.jobClass == "compare") {
+      if (this.jobTypeParams["compare"] == "expected and actual performance") {
+        // Single set of "data parameters" for both expected and actual performance
+        return ["expected and actual"];
+      } else if (
+        this.jobTypeParams["compare"] == "predicted and actual performance"
+      ) {
+        return ["predicted", "actual"];
+      } else {
+        return ["predicted", "expected"];
```

Comment:
```suggestion
        return ["predicted", "actual"];
```
will provide actual weather to compute expected performance

Reply:
but maybe that will add the performance questions which are irrelevant in this case. "actual weather" would be ok too, or worry about it later when it's unhidden

Reply:
With the `<slot>` header override added, we can adjust the name displayed for the user, and I agree that "actual weather" makes more sense there. I took a quick look at #171 and it looks like this job type has an `expected_data_params` property, so I think at least for the model/control portion of this code using the actual/expected/predicted terminology makes sense and if we need to adjust the front end that can be done with templates or a getter for the text to display to the user.